packages/ui/src/components/va-input/components/VaInputWrapper.vue

```diff
@@ -36,6 +36,12 @@
                 :style="{ color: colorComputed }"
               >
                 {{ label }}
+                <span
```

Comment:
Would be nice to have a slot here. I want required-mark to be `🐱` instead of `*` :)